lib/core/seleniumRunner.js

```diff
@@ -426,8 +426,31 @@ class SeleniumRunner {
    * @param {*} script - the script
    * @param {boolean} isAsync - is the script synchrously or async?
    * @param {*} name - the name of the script
+   * @param {[String]} requires - the requirements for executing the script
    */
-  async runScriptFromCategory(script, isAsync, name) {
+  async runScriptFromCategory(script, isAsync, name, requires) {
+    const privilegeWanted = requires.privilege;
```

Comment:
`requires && requires.privilege`, so that callers don't have to upgrade.

Reply:
Good eyes!